Bottom: 58424267ac79a08264f4a2fdac7780d19de5ad93
Top:    965567e6c2b178178e3e535d0a7327ee4a570a81
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-10 17:16:17 +0000

Revert "ActionDisplay: show help on failure."

This reverts commit 0978828fcf27be98d4edee22e57a13756a684b62.


We need to be strict about argument checking.


---

diff --git a/src/action.c b/src/action.c
index 0be31ab..58adeb6 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2961,13 +2961,11 @@ ActionDisplay (int argc, char **argv, Coord childX, Coord childY)
 	  break;
 	}
     }
-  else
-    err = 1;
 
-  if (err)
-    AFAIL (display);
+  if (!err)
+    return 0;
 
-  return 0;
+  AFAIL (display);
 }
 
 /* --------------------------------------------------------------------------- */
